### PR TITLE
History

### PR DESCRIPTION
**Pull request by vishalg8454**
This PR adds the functionality of history in VideoLibrary app. 

The redux-slices added are-
- `historySlice` at src/features/historySlice.js

The pages added are - 
- `HistoryPage` at src/pages/history-page

[live preview](https://deploy-preview-8--videostock.netlify.app)

Would love to hear your feedback and suggestions.

image

image

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -6,3 +6,4 @@ export { PlaylistPage } from "./playlist-page/PlaylistPage";
 export { PlaylistDetailPage } from "./playlist-detail-page/PlaylistDetailPage";
 export { LikePage } from "./like-page/LikePage";
 export { WatchLaterPage } from "./watch-later-page/WatchLaterPage";
+export { HistoryPage } from "./history-page/HistoryPage";
```
Comment (syed-shaida-hussain) [inline src/pages/index.js:9]: You can export similar files in one file and import them in a single line.

**Review by syed-shaida-hussain — APPROVED** [review]
Everthing looks good. You can merge it.

File: src/pages/index.js (hunk 1 above)
Comment (vishalg8454) [reply src/pages/index.js:9]: That is what I seem to be doing. Can you please check once again?

**Review by shashikantmaurya0007 — APPROVED** [review]
Code looks good to be merged

**Pull request merged**